presentation/opentelemetry/src/presentation-opentelemetry/Diagnostics.ts

```diff
@@ -0,0 +1,91 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Core
+ */
+import { Diagnostics, DiagnosticsLogEntry, DiagnosticsScopeLogs } from "@itwin/presentation-common";
+import { randomBytes } from "crypto";
```

Comment:
I don't see a dependency for `crypto`. Where should it come from?

Reply:
As far as I understand, it comes with Node.js and doesn't need to be installed separately.

Reply:
The `presentation-opentelemetry` package doesn't state that it requires backend or Node dependencies and I hope we can keep it that way (frontend doesn't have Node). Any chance we could use this function using another dependency?